app/navigation/nonsipp/LandOrPropertyNavigator.scala

```diff
@@ -163,7 +154,10 @@ object LandOrPropertyNavigator extends JourneyNavigator {
         .onPageLoad(srn, index, NormalMode)
 
     case LandOrPropertyTotalIncomePage(srn, index) => //27j5
-      controllers.routes.UnauthorisedController.onPageLoad()
+      controllers.nonsipp.landorproperty.routes.LandOrPropertyCYAController.onPageLoad(srn, index, CheckOrChange.Check)
+
+    case LandOrPropertyCYAPage(srn) =>
+      controllers.nonsipp.landorproperty.routes.LandOrPropertyListController.onPageLoad(srn, NormalMode)
```

Comment:
I assume checkRoutes below at the end of this file is the one that needs to be populated as a part of the remaining navigation issues (as a separate ticket), right?